### PR TITLE
Styles and cleanup

### PR DESCRIPTION
**Pull request by jotabraham**
no drastic changes, just some simple styling and minor js functionality

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -138,6 +138,8 @@ productContainer.addEventListener("click", (e) => {
     let index = e.target.getAttribute("data-index");
     myCartArray.push(productArray[index].price, productArray[index].name);
     console.log(myCartArray);
+    e.target.textContent = "Item Added to Cart!";
+    e.target.style.backgroundColor = "#625133";
```
Comment (HannonN) [inline script.js:142]: I like this! Very clever and simple!

File: style.css [hunk 2]
```diff
@@ -81,15 +81,20 @@ html {
...
-  padding: 5px;
+  padding: 10px 5px;
   background-color: rgba(3, 122, 122, 0.723);
   color: white;
+  border: 0px;
+  border-radius: 10px;
+}
+
+.cart-button:hover {
+  background-color: rgba(3, 122, 122, 1);
```
Comment (HannonN) [inline style.css:97]: I'm good with all of this!

File: script.js (hunk 1 above)
Comment (pskuthe30) [reply script.js:142]: That's very neat.

File: style.css [hunk 3]
```diff
@@ -13,12 +13,14 @@ html {
   display: flex;
   justify-content: space-between;
   align-items: center;
+  padding: 5px 10px;
```
Comment (HannonN) [inline style.css:16]: This is good

File: style.css (hunk 2 above)
Comment (pskuthe30) [reply style.css:97]: Let's merge it then!! what Say?

**Pull request merged**